### DISCUSSION TITLE
Create event on percentage of track played

### DISCUSSION THREAD

**Discussion by antonellogatto** [katspaugh/wavesurfer.js#3036] [Q&A]
I need a way to execute a function when a % ok the track is played.
I already made a sort of this creating an array of played intervals and updating it on any play and pause. The con is the the UI is updated when the user click on pause. I need a way to do it when the % is reached while playing the track. Is it ever possible?

**Comment by katspaugh** [comment]
You can use the timeupdate or audioprocess event and divide the current time by total duration.
